app/client/components/delivery/records/deliveryRecordsProblemReview.tsx

```diff
@@ -55,17 +55,11 @@ export const DeliveryRecordsProblemReview = (
   const renderReviewDetails = (
     potentialHolidayStopsResponseWithCredits: PotentialHolidayStopsResponse
   ) => {
-    const totalCreditAmount = potentialHolidayStopsResponseWithCredits
-      .potentials.length
-      ? Math.abs(
-          potentialHolidayStopsResponseWithCredits.potentials
-            .flatMap<number>(x => [x.credit as number])
-            .reduce(
-              (accumulator, currentValue) =>
-                accumulator + Math.abs(currentValue)
-            )
-        )
-      : 0;
+    const totalCreditAmount: number =
+      potentialHolidayStopsResponseWithCredits.potentials.length &&
+      potentialHolidayStopsResponseWithCredits.potentials
+        .flatMap(x => [Math.abs(x.credit || 0)])
```

Comment:
moved the `Math.abs` into the `flatMap` to avoid having it round everything, and also because if one doesn't pass a 'starting value' to a `reduce` it use the first value and hence it was adding a negative to a positive, resulting in an incorrect total credit amount (e.g. `-2.89 + Math.abs(-2.89` = `0`)
_really nasty bug_